### DISCUSSION TITLE
4️⃣ Wrap-up: Most interesting things

### DISCUSSION THREAD

**Discussion by crazy4pi314** [crazy4pi314/scipy-vscode-tutorial#45] [Q&A]
There is only so much that we can cover in this 4h session, and there is so much VS Code can do! We love iterating on this format, and would love to know what we need more of! Please drop a comment below for what your liked the most, and what you would like to learn more about!

**Comment by anzelpwj** [comment]
I learned a lot, thanks! My only request would be to spend more time on section 2.2 (but I know things got a bit disorganized with the computer issue).

**Comment by bjcubsfan** [comment]
My favorite part was learning about the relevant Python extensions.

**Comment by jfishe27** [comment]
I always find the Python environments to be the most difficult to manage and maintain. Also, I have concerns about malware infiltrating packages. Are Azure artifact packages scanned for malware? Anyone know any good articles on this topic?

**Reply by jfishe27** [reply]
Thanks for the nice VS Code talk today from the 3 speakers!
